R/serializer-csv.R

```diff
@@ -0,0 +1,17 @@
+#' @rdname serializers
+#' @param ... extra arguments supplied to respective internal serialization function.
+#' @export
+serializer_csv <- function(...) {
+  function(val, req, res, errorHandler) {
```

Comment:
Just inside `serializer_csv`, can we check for `readr` before creating the serializer function?

Something close to ...

```r
if (!requireNamespace("readr", quietly = TRUE)) {
  stop("`readr` must be installed for `serializer_csv` to work")
}
```

----------

Helps catch the error at build time, rather than at run time.